enterprise/frontend/src/embedding-sdk/cli/constants/database.ts

```diff
@@ -0,0 +1,43 @@
+/** Show popular database engines in the CLI */
+export const CLI_SHOWN_DB_ENGINES = [
+  "postgres",
+  "mysql",
+  "sqlserver",
+  "bigquery-cloud-sdk",
+  "snowflake",
+  "redshift",
+  "mongo",
+  "athena",
+];
+
+/** Database connection fields that are shown in the CLI */
+export const CLI_SHOWN_DB_FIELDS = [
```

Comment:
We avoid showing the non-essential database fields, and only show what is strictly necessary to connect to the user's database.